evm/src/util.rs

```diff
@@ -179,7 +176,6 @@ pub(crate) fn biguint_to_le_limbs(x: BigUint) -> Vec<u128> {
         .collect()
```

Comment:
I'd add a note that there's a big optimization opportunity in this function if need be: `digits` should be an iterator, not a vector. This function should ideally perform one malloc, whereas currently it does at least two, maybe three.